src/index.html

```diff
@@ -19,4 +21,5 @@
     </table>
     <script type="text/javascript" src="scripts/main.js"></script>
   </body>
-</html>
+
+</html>
```

Comment:
```suggestion
</html>

```
Потрібно додавати пустий рядок в кінці коду